analysis/cnvs/prep_clinvar_cnvs.ipynb

```diff
@@ -212,29 +207,57 @@
    "cell_type": "code",
    "execution_count": 11,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "<>:44: SyntaxWarning: invalid escape sequence '\\d'\n",
+      "<>:44: SyntaxWarning: invalid escape sequence '\\d'\n",
+      "/var/folders/_d/tfjn_dfn6wl35y6ywjvyxlhh0000gq/T/ipykernel_40243/2901792136.py:44: SyntaxWarning: invalid escape sequence '\\d'\n",
+      "  out = re.sub(\"\\d+\", \"#\", out)\n"
+     ]
+    }
+   ],
    "source": [
     "### errors stored as a list of values, some of which are strings and other of which are dictionaries (determined by whether error was handled at the level of Variation Normalizer or after the normalizer)\n",
     "### this function extracts the text error responses for better readability and ease string processing\n",
     "\n",
+    "import ast\n",
     "\n",
     "def get_errors(errors: list) -> str:\n",
```

Comment:
So it looks like errors could be a list, string, or null (might be wrong)? But will want to update the type

Reply:
This also looks like it's being used in both clinvar_variation_analysis + prep_clinvar_cnvs. We should move it to the utils so it's only defined once